service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/DocumentBundle.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.fpl.model.common;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor(onConstructor_ = {@JsonCreator})
+public class DocumentBundle {
+    private final DocumentReference document;
```

Comment:
i would maybe call it documentReference or something :)

Reply:
CCD generates the 'document' tag when used in a collection. We already have the 'DocumentReference' model object.

Reply:
👍